lib/types/store.ex

```diff
@@ -137,4 +138,15 @@ defmodule Types.Store do
       {:error, :not_found} -> store
     end
   end
+
+  def fetch_store do
```

Comment:
I would put this in `LambdaEthereumConsensus.Store.StoreStore` do we don't overload this module. And we should rename (DB) Store to Storage or something xD